### ISSUE TITLE
Images with url "file://" are detected as "remote urls"

### ISSUE THREAD

**Issue by MatthiasKuehneEllerhold** [mpdf/mpdf#1887]
### Guidelines

- [X] I understand that [if I fail to provide all required details, this issue may be closed without review](https://github.com/mpdf/mpdf/blob/development/.github/CONTRIBUTING.md).

### Description of the bug

Using "file:///var/www/public/test.png" als an URL in an "<image />" tag results in a broken image in the pdf.

Version 8.0.17 works with "file://" links.
Version 8.1.6 works with just "/var/www/public/test.png"

This error is similar to https://github.com/mpdf/mpdf/issues/1646 but still different enough for its own issue report.
It seems [@JanSlabon ](https://github.com/jonsa) had the same error in comment https://github.com/mpdf/mpdf/issues/1646#issuecomment-1568836727

### mPDF version

8.1.6

### PHP Version and environment (server type, cli provider etc., enclosing libraries and their respective versions)

PHP 8.2

### Reproducible PHP+CSS+HTML snippet suffering by the error

```php
$html = '<img src="file:///var/www/public/test.png" />';

$mpdf = new Mpdf(
  [
    'mode'        => 'utf-8',
    'orientation' => 'P',
    'format'      => 'A4-P',
    'tempDir'     => '/temp/mpdf',
    'img_dpi'     => 300,
  ]
);

$mpdf->WriteHTML($html);
$pdf = $mpdf->Output('', 'S');
```

**Comment by MatthiasKuehneEllerhold** [comment]
This workaround can be used:
```php
$html = str_replace(
  'file://',
  '',
  $html
);

$mpdf->WriteHTML($html);
```

**Comment by JanSlabon** [comment]
> It seems [@JanSlabon ](https://github.com/jonsa) had the same error in comment [#1646 (comment)](https://github.com/mpdf/mpdf/issues/1646#issuecomment-1568836727)

Why I'm tagged, but another person is linked?

**Comment by machour** [comment]
@finwe ran into this today, the problem is in `src/AssetFetcher.php`'s `isPathLocal` that needs to take `file://` into account

```diff
	public function isPathLocal($path)
	{
+        if (strpos($path, "file://") === 0) return true;
		return strpos($path, '://') === false; // @todo More robust implementation
	}
```

**Comment by finwe** [comment]
file:// check quickfix implemented and released in 8.2.0. Create a separate issue if this causes other issues.
